rabbitmq/ci/start-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source utils/bash/ci.sh
+
+NAME='dd-test-rabbitmq'
+
+stop-docker $NAME
```

Comment:
Should we do this? In the weird case where a customer could have a container up and running intentionally with the same name (I know very weird) we shouldn't stop that for him without asking. I maybe would just skip this and let him kill it manually? Test would fail on the `docker run`, but it shouldn't fail on Travis.